interfaces/api/v1/world/bible.py

```
@@ -199,29 +192,35 @@ async def _generate_task():
-            loc_desc = "、".join(c['name'] for c in locs[:3])
-            bible_summary = f"主要角色：{char_desc}。重要地点：{loc_desc}。文风：{bible_data.get('style', '')}。"
-
-            # 生成初始 Knowledge
-            await knowledge_generator.generate_and_save(
-                novel_id,
-                novel.title,
-                bible_summary
-            )
-            logger.info(f"Bible and Knowledge generated successfully for {novel_id}")
-            clear_bible_generation_state(novel_id)
+            # 只在世界观生成阶段生成一次知识，避免重复调用
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)